src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderProgramProfile.kt

```diff
@@ -46,10 +51,27 @@ data class OffenderProgramProfile(
   val prison: AgencyLocation? = null,
 
   @Column(name = "OFFENDER_END_DATE")
-  val endDate: LocalDate? = null,
+  var endDate: LocalDate? = null,
 
   @OneToMany(mappedBy = "offenderProgramProfile", cascade = [CascadeType.ALL], fetch = FetchType.LAZY)
   val payBands: MutableList<OffenderProgramProfilePayBand> = mutableListOf(),
+
+  @ManyToOne
+  @NotFound(action = NotFoundAction.IGNORE)
```

Comment:
Do we always do this if there is no constraint? Would it break Nomis if we had a code here that isn't on the reference domain?